<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerSettings</class>
 <widget class="QDialog" name="CustomerSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>505</width>
    <height>520</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Customer Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/kabtrak.ico</normaloff>images/kabtrak.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="keybrd" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/connection_svg.svg</normaloff>:/images/images/connection_svg.svg</iconset>
      </attribute>
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="verticalSpacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="lblCompanyName">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Company Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtCompanyName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblAddress">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Address:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtAddress1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Address line 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtAddress2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-left-color: rgb(0, 0, 0);
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(0, 0, 0);</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Address line 2</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtAddress3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Address line 3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblContactPerson">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Contact Person:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtContactPerson">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>Contact person</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblContactNumber">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Contact Number:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtContactNumber">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblEmail">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Email:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtEmail">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>someone@company.com</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTail">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QGroupBox" name="gbOptions">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Other Options</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="LOGO">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>81</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>281</width>
                <height>81</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnLoadLogo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>281</width>
                <height>81</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Century Gothic</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Load your Logo</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <item row="4" column="0">
             <widget class="QCheckBox" name="cbbatchNumber">
              <property name="font">
               <font>
                <family>Century Gothic</family>
                <pointsize>9</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>User Batch</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="cbWorkOrderNumeric">
              <property name="text">
               <string>Numeric</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="cbWorkOrder">
              <property name="font">
               <font>
                <family>Century Gothic</family>
                <pointsize>9</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>User Work  Order</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QCheckBox" name="cbBatchNumeric">
              <property name="text">
               <string>Numeric</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Min</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sbWorkOrderMin">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Max</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sbWorkOrderMax">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="6" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Min</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sbBatchMin">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="6" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Max</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sbBatchMax">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>120</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/document_text.svg</normaloff>:/images/images/document_text.svg</iconset>
      </attribute>
      <attribute name="title">
       <string>Notes</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="mmoNotes">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="plainText">
          <string>qWQWQW</string>
         </property>
         <property name="placeholderText">
          <string>Notes Here</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabReg">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/note_text-edit.svg</normaloff>:/images/images/note_text-edit.svg</iconset>
      </attribute>
      <attribute name="title">
       <string>Registration/Branding</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
